src/main/java/com/hobbycollection/HobbyCollectionRESTController.java

```diff
@@ -10,14 +12,19 @@
     @Autowired
     private ICollectionService collectionService;
 
+    Logger log = LoggerFactory.getLogger(this.getClass());
+
```

Comment:
I added code for getting a logger.
Sources:
https://www.youtube.com/watch?v=cMVj0yF5x0Q